overlord/devicestate/handlers_install.go

```diff
@@ -854,12 +854,12 @@ func mountSeedSnap(seedSn *seed.Snap) (mountpoint string, unmount func() error,
 }
 
 func (m *DeviceManager) loadAndMountSystemLabelSnaps(systemLabel string) (
-	*System, map[snap.Type]*snap.Info, map[snap.Type]*seed.Snap, map[snap.Type]string, func(), error) {
+	*System, seed.Seed, map[snap.Type]*snap.Info, map[snap.Type]*seed.Snap, map[snap.Type]string, func(), error) {
```

Comment:
then we could give it the same treatment or just reduce to 3 returned values, your call

```suggestion
	sysInfo *systemAndEssentialSnaps, mountPoints map[snap.Type]string, func(), err error) {
```

Reply:
I think here 3 is probably not too bad.